api/nakadi-event-bus-api.yaml

```diff
@@ -317,7 +345,7 @@ paths:
           required: true
         - name: batch_limit
           in: query
-          description: Maximum number of `SimpleStreamElement`s in each chunk of the stream. If unspecified assumes default value 1 (i.e. each event is individually submitted).
+          description: Maximum number of `EventStreamAtom`s in each chunk of the stream. If unspecified assumes default value 1 (i.e. each event is individually submitted).
```

Comment:
Actually I think we have some problem here. @v-stepanov what does this parameter do in python prototype? Are we talking about number of `Event`s in `EventStreamAtom.events` array?